thirdparty_build.gradle

```diff
@@ -9,8 +9,25 @@ buildscript {
 /*DJVU CHAIN*/
 task downloadDjvu {
     doLast {
```

Comment:
Why do you put new exec instuction in `doLast` and kept 
```
        exec {
            workingDir file(djvu)
            commandLine = ['git', 'checkout', '32ff0948c5bc27c0b7507dba90e726fc941fcc42'/*current master*/]
        }
 ```
 out of it?

Reply:
Here I am trying to handle the edge case if we are trying to checkout the commit later then the one known currently checked out repository. So we first move HEAD by fetching the latest master and then switching to the pinned commit. So we either clone or fetch master before switching to detached HEAD.